src/main/interface/msp.c

```diff
@@ -559,12 +560,14 @@ static bool mspCommonProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProce
         sbufWriteU16(dst, batteryConfig()->batteryCapacity); // in mAh
 
         // battery state
-        sbufWriteU8(dst, (uint8_t)constrain((getBatteryVoltage() / 10), 0, 255)); // in 0.1V steps
+        sbufWriteU8(dst, (uint8_t)constrain((getBatteryVoltage() + 5) / 10, 0, 255)); // in 0.1V steps
         sbufWriteU16(dst, (uint16_t)constrain(getMAhDrawn(), 0, 0xFFFF)); // milliamp hours drawn from battery
         sbufWriteU16(dst, (int16_t)constrain(getAmperage(), -0x8000, 0x7FFF)); // send current in 0.01 A steps, range is -320A to 320A
 
         // battery alerts
         sbufWriteU8(dst, (uint8_t)getBatteryState());
+		
+        sbufWriteU16(dst, (uint16_t)constrain(getBatteryVoltage(), 0, 0xFFFF)); // in 0.01V steps
```

Comment:
No need for the `constrain()` here - `getBatteryVoltage()` is already an `uint16_t`.